crates/sui-indexer/src/indexer.rs

```diff
@@ -86,17 +88,23 @@ impl Indexer {
             .unwrap_or(CHECKPOINT_PROCESSING_BATCH_DATA_LIMIT.to_string())
             .parse::<usize>()
             .unwrap();
+        let extra_reader_options = ReaderOptions {
```

Comment:
ah, i guess this is called `ReaderOptions` because it configures how the indexer writer reads the data from the fullnode and/ or checkpoint snapshots?

Reply:
Yup!